tests/test_circuit_validation.py

```diff
@@ -531,253 +385,59 @@ def test_no_template_library_files():
-            del h5f["edges/default/edge_group_index"]
-            del h5f["edges/default/edge_group_id"]
+            del h5f[to_remove]
         errors = validate(str(config_copy_path))
         assert errors == set()
 
 
-def test_edge_population_edge_group_different_length():
+def test_no_edge_indices():
     with copy_test_data() as (circuit_copy_path, config_copy_path):
         edges_file = circuit_copy_path / "edges.h5"
         with h5py.File(edges_file, "r+") as h5f:
-            del h5f["edges/default/edge_group_index"]
-            h5f.create_dataset("edges/default/edge_group_index", data=[0, 1, 2, 3, 4])
+            del h5f["edges/default/indices"]
         errors = validate(str(config_copy_path))
         assert errors == {
-            Error(
-                Error.FATAL,
-                'Population {} of {} has different sizes of "group_id" and "group_index"'.format(
-                    "/edges/default", edges_file
-                ),
-            )
-        }
-
-
-def test_edge_population_wrong_group_id():
-    with copy_test_data() as (circuit_copy_path, config_copy_path):
-        edges_file = circuit_copy_path / "edges.h5"
-        with h5py.File(edges_file, "r+") as h5f:
-            del h5f["edges/default/edge_group_id"]
-            h5f.create_dataset("edges/default/edge_group_id", data=[0, 1, 0, 0])
-        errors = validate(str(config_copy_path))
-        assert errors == {
-            Error(
-                Error.FATAL,
-                "Population /edges/default of {} misses group(s): {}".format(edges_file, {1}),
-            )
-        }
-
-
-def test_edge_population_ok_group_index():
-    with copy_test_data() as (circuit_copy_path, config_copy_path):
-        edges_file = circuit_copy_path / "edges.h5"
-        with h5py.File(edges_file, "r+") as h5f:
-            del h5f["edges/default/edge_group_id"]
-            del h5f["edges/default/edge_group_index"]
-            h5f.create_group("edges/default/1")
-            h5f.create_dataset("edges/default/1/test", data=[1, 1])
-            h5f.create_dataset("edges/default/edge_group_id", data=[0, 1, 0, 1])
-            h5f.create_dataset("edges/default/edge_group_index", data=[0, 0, 1, 1])
-        errors = validate(str(config_copy_path))
-        assert errors == set()
-
-
-def test_edge_population_wrong_group_index():
-    with copy_test_data() as (circuit_copy_path, config_copy_path):
-        edges_file = circuit_copy_path / "edges.h5"
-        with h5py.File(edges_file, "r+") as h5f:
-            del h5f["edges/default/edge_group_index"]
-            h5f.create_dataset("edges/default/edge_group_index", data=[0, 1, 2, 12])
-        errors = validate(str(config_copy_path))
-        assert errors == {
-            Error(
-                Error.FATAL,
-                "Group default/0 in file {} should have ids up to {}".format(edges_file, 12),
-            )
-        }
-
-
-def test_edge_population_wrong_group_index_multi_group():
```

Comment:
are the tests that are deleted covered by other things or not relevant any more?